### ISSUE TITLE
Support template literals

### ISSUE THREAD

**Issue by tpict** [seleb/eslint-plugin-no-smart-quotes#19]
It seems that smart quotes aren't detected if they're inside a template literal e.g.
```ts
`“`
```

Is it possible to add support for these?

Thanks for the plugin, going to spare me a lot of grief at work.
